flashlight/memory/MemoryManagerAdapter.cpp

```diff
@@ -14,12 +14,12 @@
 #include "flashlight/common/Utils.h"
 
 namespace fl {
-
+// note: 'interface' is a define/keyword for MSVC, not to be used for var names
```

Comment:
nit: remove comment - putting it in the commit description is fine :)

Reply:
done